fdbserver/workloads/MetaclusterManagementConcurrencyWorkload.actor.cpp

```diff
@@ -48,9 +48,11 @@ struct MetaclusterManagementConcurrencyWorkload : TestWorkload {
 	std::vector<ClusterName> dataDbIndex;
 
 	double testDuration;
+	bool enableMetaclusterTenantModeCheck;
```

Comment:
Do we need to make this configurable? It could be ok to have the tests just pass false.